File: Modules/HumanResource/TrainingManagement/MasterProgram/MasterProgramRoutineType.cs
using System.ComponentModel;

namespace HRMSoftware.TrainingManagement;
[Serenity.ComponentModel.EnumKey("HumanResource.TrainingManagement.MasterProgramRoutineType")]
public enum MasterProgramRoutineType
{
    [Description("No Repeat")]
    No = 0,
    [Description("daily")]
    Daily = 1,
    [Description("Weekly")]
    Weekly = 2,
    [Description("Monthly")]
    Monthly = 3,
    [Description("Yearly")]
    Yearly = 4,
}

File: Modules/ServerTypes/TrainingManagement/MasterProgramRoutineType.ts
import { Decorators } from "@serenity-is/corelib";

export enum MasterProgramRoutineType {
    No = 0,
    Daily = 1,
    Weekly = 2,
    Monthly = 3,
    Yearly = 4
}
Decorators.registerEnumType(MasterProgramRoutineType, 'HRMSoftware.TrainingManagement.MasterProgramRoutineType', 'HumanResource.TrainingManagement.MasterProgramRoutineType');